.travis.yml

```diff
@@ -0,0 +1,60 @@
+language: scala
+
+sudo: required
+
+dist: trusty
+
+# Install Docker on Travis.  Merely exposing Docker through the Travis
+# settings is insufficient, because the docker-compose tests we use are
+# unable to see it in that case.
+install:
+  - curl -sSL "https://get.docker.com/gpg" | sudo -E apt-key add -
+  - echo "deb https://apt.dockerproject.org/repo ubuntu-trusty main" | sudo tee -a /etc/apt/sources.list
+  - sudo apt-get update
+  - sudo apt-get -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" --assume-yes install docker-engine
+  - docker version
+
+cache:
+  directories:
+    - "~/.sbt"
+    - "~/.ivy2"
+    - "project/target/resolution-cache"
+    - "project/target/streams"
+    - "api/target"
+    - "calm_adapter/target"
+    - "common/target"
+    - "id_minter/target"
+    - "ingestor/target"
+    - "transformer/target"
+
+matrix:
+  include:
+    - scala: 2.12.2
+      env: PROJECT=common
+    - scala: 2.12.2
+      env: PROJECT=api
+    - scala: 2.12.2
+      env: PROJECT=transformer
+    - scala: 2.12.2
+      env: PROJECT=calm_adapter
+    - scala: 2.12.2
+      env: PROJECT=ingestor
+    - scala: 2.12.2
+      env: PROJECT=id_minter
```

Comment:
> Do we need to specify it in the travis.yaml as well since it's defined in our sbt build? Can we not avoid the repetition?

Maybe not, I still see 2.11.8 in several places in the build output.

> that seems to be run within platform-infra which is not an sbt project, so it doesn't have any scalaVersion defined

Oops, yep.